autoscheduler/frontend/src/components/HelpText.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Paper } from '@material-ui/core';
+
+const style = {
+  height: 350,
+  width: 650
+}
+
+const centerText = {
+  flex: 1,
+  justifyContent: 'center',
+  alignItems: 'center'
+}
+
+const HelpText: React.SFC = function App() {
+  return (
+
+    <Grid
+    container
+    spacing={0}
+    direction="row"
+    justify="space-around"
+    alignItems="center"
+    style={{ minHeight: '75vh' }}>
+
+      <Paper style={style}>
+        <h1>
+          <Grid
+          container
+          spacing={0}
+          direction="row"
+          justify="space-around"
+          alignItems="center">
+            Guidelines
+          </Grid>
+        </h1>
+        <h2 text-align = "center">
+        <div
+        style={{
```

Comment:
Once I add style-loader, we should convert this to using CSS modules. That way, the TSX files can handle logic and the CSS files can handle styling. The buzzword for this is "separation of concerns."